src/client/actions/MessageReactionRemoveEmoji.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const Action = require('./Action');
+const { Events } = require('../../util/Constants');
+
+class MessageReactionRemoveEmoji extends Action {
+  handle(data) {
+    const channel = this.getChannel(data);
+    if (!channel || channel.type === 'voice') return false;
+
+    const message = this.getMessage(data, channel);
+    if (!message) return false;
+
+    const reaction = this.getReaction(data, message);
+    if (!reaction) return false;
```

Comment:
```suggestion
    if (!reaction) return false;
    message.reactions.delete(reaction.emoji.id || reaction.emoji.name);
```
You should delete the reaction from cache

Reply:
👍